changelog/4616.breaking.rst

```diff
@@ -0,0 +1,6 @@
+The following `~sunpy.map.Map` methods have had support for specific positional
+arguments removed. They must now be passed as keyword arguments
+(i.e. ``m.method(keyword_arg=value)``).
+
+- :meth:`~sunpy.map.Map.submap`: ``width``, ``height``.
+- :meth:`~sunpy.map.Map.draw_rectangle`: ``width``, ``height``, ``axes``, ``top_right``.
```

Comment:
does our changelog rendering actually respect whitespace like this?

Reply:
Only one way to find out.

Reply:
the line break isn't rendered but the list is.